service/src/main/java/uk/gov/hmcts/reform/fpl/model/Proceeding.java

```diff
@@ -21,29 +29,25 @@
     private final String guardian;
     private final String sameGuardianNeeded;
     private final String sameGuardianDetails;
+    private final List<Element<OtherProceeding>> additionalProceedings;
 
-    @JsonCreator
-    public Proceeding(@JsonProperty("onGoingProceeding") final String onGoingProceeding,
-                      @JsonProperty("proceedingStatus") final String proceedingStatus,
-                      @JsonProperty("caseNumber") final String caseNumber,
-                      @JsonProperty("started") final String started,
-                      @JsonProperty("ended") final String ended,
-                      @JsonProperty("ordersMade") final String ordersMade,
-                      @JsonProperty("judge") final String judge,
-                      @JsonProperty("children") final String children,
-                      @JsonProperty("guardian") final String guardian,
-                      @JsonProperty("sameGuardianNeeded") final String sameGuardianNeeded,
-                      @JsonProperty("sameGuardianDetails") final String sameGuardianDetails) {
-        this.onGoingProceeding = onGoingProceeding;
-        this.proceedingStatus = proceedingStatus;
-        this.caseNumber = caseNumber;
-        this.started = started;
-        this.ended = ended;
-        this.ordersMade = ordersMade;
-        this.judge = judge;
-        this.children = children;
-        this.guardian = guardian;
-        this.sameGuardianNeeded = sameGuardianNeeded;
-        this.sameGuardianDetails = sameGuardianDetails;
+    @JsonIgnore
+    public OtherProceeding getFirstProceeding() {
+        if (StringUtils.isNotEmpty(onGoingProceeding)) {
+            return OtherProceeding.builder()
+                .onGoingProceeding(this.getOnGoingProceeding())
+                .proceedingStatus(this.getProceedingStatus())
+                .caseNumber(this.getCaseNumber())
+                .started(this.getStarted())
+                .ended(this.getEnded())
+                .ordersMade(this.getOrdersMade())
+                .judge(this.getJudge())
+                .children(this.getChildren())
+                .guardian(this.getGuardian())
+                .sameGuardianNeeded(this.getSameGuardianNeeded())
+                .sameGuardianDetails(this.getSameGuardianDetails())
+                .build();
+        }
```

Comment:
return this?

Reply:
or if a clone is required, use the toBuilder method

Reply:
The json format of proceeding is 

`"proceeding": {
      "ended": "02-02-2006",
      "judge": "William Peters",
      "started": "02-02-2002",
      "children": "children subject to proceedings",
      "guardian": "Mark Watson",
      "caseNumber": "12345",
      "ordersMade": "Supervision order",
      "proceedingStatus": "Ongoing",
      "onGoingProceeding": "Yes",
      "sameGuardianNeeded": "No",
      "sameGuardianDetails": "Some guardian not needed",
      "additionalProceedings": [
        {
          "id": "a5e57424-9b5c-4722-be7e-9d9d9111c2b9",
          "value": {
            "ended": "02-02-2009",
            "judge": "Peters Williams",
            "started": "02-02-2008",
            "children": "children subject to proceedings",
            "guardian": "John Watson",
            "caseNumber": "12345",
            "ordersMade": "Supervision order",
            "proceedingStatus": "Ongoing",
            "onGoingProceeding": "Yes",
            "sameGuardianNeeded": "No",
            "sameGuardianDetails": "Some guardian not needed"
          }
        }
      ]
    },`

but i want it to be like 

proceeding {
    firstOtherProceeding,
     List<OtherProceedings>
} so that i can process all the proceedings easily, so added this